### PR TITLE
15.3.0

### PR DESCRIPTION
**Pull request by nehabluedot**
Can be tested after adding a valid ProjectID

### CODE REVIEW

File: BDPointAndroidXamarinDemo/MainActivity.cs [hunk 1]
```diff
@@ -85,84 +49,135 @@ protected override void OnCreate(Bundle savedInstanceState)
...
-            }
+            GeoTriggeringService.Builder()
+               .InvokeNotification(CreateNotification())
+               .Start(this, this);
+        }
+
+        private void StartBgGeoTrigger()
+        {
+            GeoTriggeringService.Builder()
+                .Start(this, this);
```
Comment (jsprincep) [inline BDPointAndroidXamarinDemo/MainActivity.cs]: We should not be using 'this' for the result receiver, it should be inline.

File: BDPointAndroidXamarinDemo/MainActivity.cs [hunk 2]
```diff
@@ -230,10 +245,47 @@ private Notification createNotification()
             }
         }
 
+        public void OnInitializationFinished(BDError error)
```
Comment (jsprincep) [inline BDPointAndroidXamarinDemo/MainActivity.cs]: This should not be a member function.

**Review by jsprincep — CHANGES_REQUESTED** [review]

File: BDPointAndroidXamarinDemo/MainActivity.cs (hunk 2 above)
Comment (nehabluedot) [reply BDPointAndroidXamarinDemo/MainActivity.cs]: moved this to a separate class

File: BDPointAndroidXamarinDemo/MainActivity.cs (hunk 1 above)
Comment (nehabluedot) [reply BDPointAndroidXamarinDemo/MainActivity.cs]: I tried inline but was not able to get it instead I have created separate listeners, please check and suggest if any other improvement can be done

**Review by jsprincep — APPROVED** [review]

**Pull request merged**